core/kazoo_im/src/kz_im.erl

```diff
@@ -38,15 +34,13 @@
         ,remove_originator_flag/2
         ]).
 
--export([type/0, type/1]).
-
--include("kz_documents.hrl").
+-include("kazoo_im.hrl").
 
 -type payload() :: kz_json:object().
 -export_type([payload/0]).
```

Comment:
`set_body/2` asks for a `kz_term:ne_binary()` for the Body but `cb_sms:120` uses `'null'`. Can you update to use either `ne_binary() | 'null'` or `kz_json:json_term()` (which allows 'null').

Reply:
sure